src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDocletWriter.java

```diff
@@ -2249,21 +2245,6 @@ public void addPreviewInfo(Element forWhat, Content target) {
                 declaredUsingPreviewFeature.addAll(memberAPITypes.declaredUsingPreviewFeature);
                 previewAPI.addAll(memberAPITypes.previewAPI);
                 reflectivePreviewAPI.addAll(memberAPITypes.reflectivePreviewAPI);
-                previewLanguageFeatures.addAll(utils.previewLanguageFeaturesUsed(enclosed));
-            } else if (!utils.previewLanguageFeaturesUsed(enclosed).isEmpty()) {
-                declaredUsingPreviewFeature.add((TypeElement) enclosed);
-            }
-        }
-        previewLanguageFeatures.addAll(utils.previewLanguageFeaturesUsed(el));
-        if (!previewLanguageFeatures.isEmpty()) {
-            if (previewLanguageFeatures.contains(DeclarationPreviewLanguageFeatures.SEALED_PERMITS)) {
-                previewLanguageFeatures.remove(DeclarationPreviewLanguageFeatures.SEALED);
-            }
```

Comment:
These lines should go, because they specifically refer to SEALED stuff, but the rest of the block could stay